runner.go

```diff
@@ -816,9 +823,9 @@ func (r *Runner) storePid() error {
 		return nil
 	}
 
-	log.Printf("[INFO] creating pid file at %q", path)
+	namedLogger("runner").Info("creating pid file at %q", path)
 
-	f, err := os.OpenFile(path, os.O_CREATE|os.O_WRONLY|os.O_TRUNC, 0666)
+	f, err := os.OpenFile(path, os.O_CREATE|os.O_WRONLY|os.O_TRUNC, 0o666)
```

Comment:
Just wanted to double-check that the change from `0666` to `0o666` is intentional?

Reply:
I'm using 'gofumpt' as my formatter and it converted that automatically. 

The rule (looked it up)...
> Octal integer literals should use the 0o prefix on modules using Go 1.13 and later

I usually filter these out into a separate commit (so they don't confuse the PR), but this one was so small I just let it go.

Reply:
Thanks for explaining, that makes sense!